internal/updater/checker.go

```diff
@@ -29,7 +30,7 @@ type Checker struct {
 var DefaultChecker = newDefaultChecker()
 
 func newDefaultChecker() *Checker {
-	updateURL := constants.APIUpdateURL
+	updateURL := constants.APIUpdateURL + url.QueryEscape(constants.CommandName)
```

Comment:
I think it would be cleaner simply to update `APIUpdateURL` itself to have the `/state` suffix.

Reply:
Also; no need to use `constants.CommandName`. The ship has sailed on keeping that name centralized.